micro_sam/training/trainable_sam.py

```diff
@@ -15,15 +15,18 @@ class TrainableSAM(nn.Module):
     Args:
         sam: The SegmentAnything Model.
         device: The device for training.
+        upsampled_masks: Whether to return the output masks in the original input shape.
     """
     def __init__(
         self,
         sam: Sam,
         device: Union[str, torch.device],
+        upsampled_masks: bool = True,
```

Comment:
Let's remove all the changes here and just always return low res and upsampled masks.

Reply:
This is restored now!